package.json

```diff
@@ -1,40 +1,43 @@
 {
   "name": "angular-architects",
   "version": "0.0.0",
+  "private": true,
   "license": "MIT",
   "scripts": {
-    "commit": "git-cz",
-    "nx": "nx",
-    "start": "nx serve",
-    "build": "nx build ddd",
-    "pack": "npm pack ./dist/libs/ddd",
-    "build:schema": "node libs/ddd/src/schematics/json-schema-to-ts.js",
-    "test": "nx test",
-    "lint": "nx workspace-lint && nx lint",
-    "e2e": "nx e2e",
+    "affected": "nx affected",
```

Comment:
The changes to `package.json` in this commit would be easier to review if you include a commit that sorts the `scripts` entries by key name. Essentially you can apply the lint rules like this https://ota-meshi.github.io/eslint-plugin-jsonc/rules/sort-keys.html in a separate commit.